src/main/cpp/RobotContainer.cpp

```diff
@@ -21,12 +22,16 @@ RobotContainer::RobotContainer()
     )
   );
 
+  mConveyor.SetDefaultCommand(SetConveyor(mConveyor));
+
   // Configure the button bindings
   ConfigureButtonBindings();
 }
 
 void RobotContainer::ConfigureButtonBindings() {
   // Configure your button bindings here
+  frc2::Trigger( [this] { return mConveyor.IsBallComing(); }).WhenActive( [this]{ mConveyor.BallIntakeIncoming(); });
+  frc2::Trigger( [this] { return mConveyor.IsBallExiting(); }).WhenInactive( [this] { mConveyor.BallIntakeExiting(); });
```

Comment:
I think we need the complements of these buttons as well. That is we need both Active and Inactive states from both sensors.

A logic diagram for ball counting should be designed. Some test code would be useful.